src/Skeletor/Manager/ProviderManager.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Skeletor\Manager;
+
+use Skeletor\Frameworks\Framework;
+use Skeletor\Packages\Package;
+
+class ProviderManager extends Manager
+{
+    const PROVIDER_METHOD = 'getProviderClass';
+    const FACADE_METHOD = 'getFacadeClass';
+
+    /**
+     * @param Package $package
+     * @param Framework $activeFramework
+     */
+    public function configure(Package $package, Framework $activeFramework)
+    {
+        if ($activeFramework->getName() !== 'Laravel') {
+            $this->cli->red("Configure {$package->getInstallSlug()} manual for {$activeFramework->getName()}, we currently only support Laravel");
+            return;
+        }
+        $configFile = $activeFramework->getPath('appConfig');
+        $newContent = $this->getNewConfig($configFile, $package);
+        $this->projectFilesystem->update($configFile, $newContent);
+    }
+
+    /**
+     * @param string $configFile
+     * @param Package $package
+     * @return array|bool
+     */
+    public function getNewConfig(string $configFile, Package $package)
+    {
+        $state = null;
```

Comment:
I think this has to be part of the foreach body or we can have side effects depending on how the lines are ordered